ansys/mapdl/core/post.py

```diff
@@ -519,334 +521,239 @@ def plot_element_values(
             show_elem_numbering=show_elem_numbering, **kwargs
         )
 
-    def element_displacement(
-            self, component="ALL", option="AVG"
-    ) -> np.ndarray:
-        """Return element displacement.
-
-        One value per element.  Either minimum, maximum, or average of
-        all nodes in each element.
-
-        Equilvanent MAPDL commands:
-
-        * ``ETABLE,VALUES,U,X``
-        * ``PRETAB,VALUES`` or ``*VGET,TMP,ELEM,1,ETAB,VALUES``
-
-        Parameters
-        ----------
-        component : str, optional
-            Structural displacement component to retrieve.  Must be
-            ``"X"``, ``"Y"``, ``"Z"``, ``"ALL"`` or ``"NORM"``.  Defaults to
-            ``"ALL"``.
-        option : str, optional
-            Option for storing element table data.  One of the
-            following:
-
-            * ``"MIN"`` : Store minimum element nodal value of the
-              specified item component.
-            * ``"MAX"`` : Store maximum element nodal value of the
-              specified item component.
-            * ``"AVG"`` : Store averaged element centroid value of the
-              specified item component (default).
+    def _plot_point_scalars(self, scalars, show_node_numbering=False, **kwargs):
+        """Plot point scalars"""
+        if not scalars.size:
+            raise RuntimeError(
+                "Result unavailable.  Either the result has not been loaded "
+                "with ``mapdl.set(step, sub_step)`` or the result does not "
+                "exist within the result file."
+            )
 
-        Examples
-        --------
-        Return the average element displacement for all components.
+        sel = self.selected_nodes
+        all_scalars = np.zeros(sel.shape)
+        all_scalars[sel] = scalars
 
-        >>> arr = mapdl.post_processing.element_displacement("ALL")
-        >>> arr.shape
-        (2080, 3)
-        >>> arr
-        array([[ 1.07396154e-06, -9.03608033e-06, -5.17768108e-12],
-               [ 3.15631730e-06, -2.65527340e-05,  1.07714512e-11],
-               [ 5.12543515e-06, -4.31175194e-05,  2.19929719e-12],
-               ...,
-               [ 5.41204700e-06, -4.80335719e-05,  7.75819589e-11],
-               [ 3.33649806e-06, -2.96109417e-05,  1.44947535e-10],
-               [ 1.13836132e-06, -1.01038096e-05,  6.95566641e-11]])
+        surf = self._mapdl.mesh._surf
 
-        """
-        check_elem_option(option)
-        component = component.upper()
-        check_comp(component, DISP_TYPE)
+        nods = surf["ansys_node_num"]-1
```

Comment:
Nitpick: Be mindful of whitespace surrounding operators.
```suggestion
        nods = surf["ansys_node_num"] - 1
```